### PR TITLE
fix: change URLs of GitHub releases

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

- Change URLs of GitHub releases to the latest version.

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: v2/data/packages.xml [hunk 1]
```diff
@@ -937,7 +939,7 @@
 		<pageURL>https://www.nicovideo.jp/watch/sm33020019</pageURL>
 		<downloadURL
 		>https://github.com/oov/aviutl_gcmzdrops/releases/latest</downloadURL>
-		<latestVersion>v0.3.18</latestVersion>
+		<latestVersion>v0.3.25</latestVersion>
```
Comment (mitosagi) [inline v2/data/packages.xml:942]: It might be better to add it to the v1 XML as well.

**Review by mitosagi — COMMENTED** [review]
I'll check one thing.

**Comment by mitosagi** [comment]
The v1 GcmzDrops has not been updated, is it correct that we will basically only update the v2 in the future?

**Comment by hal-shu-sato** [comment]
I'd like to do so.
The changes that had been generated in #60 got mixed up in this PR.

**Review by mitosagi — APPROVED** [review]
Good change.

**Pull request merged**